unbound/metadata.csv

```diff
@@ -0,0 +1,82 @@
+unbound.thread.requestlist.max,gauge,,,,Maximum size attained by the internal recursive processing request list,1,unbound,thread.requestlist.max
+unbound.thread.requestlist.overwritten,count,,,,The number of requests in the request list that were overwritten by newer entries,1,unbound,thread.requestlist.overwritten
+unbound.thread.requestlist.exceeded,gauge,,,, Queries that were dropped because the request list was full,1,unbound,thread.requestlist.exceeded
+unbound.thread.requestlist.current.all,gauge,,,,Current size of the request list,0,unbound,thread.requestlist.current
+unbound.thread.requestlist.current.user,gauge,,,,Current size of the request list (only the requests from client queries),0,unbound,thread.requestlist.current.user
+unbound.thread.recursion.time.avg,gauge,,,,Average time it took to answer queries that needed recursive processing,0,unbound,thread.recursion.time.avg
+unbound.thread.recursion.time.median,gauge,,,,The median of the time it took to answer queries that needed recursive processing,0,unbound,thread.recursion.time.median
+unbound.thread.tcpusage,gauge,,,,The currently held tcp buffers for incoming connections,0,unbound,thread.tcpusage
+unbound.total.num.queries,count,,,,summed over threads,1,unbound,total.num.queries
+unbound.total.num.cachehits,count,,,,summed over threads,1,unbound,total.num.cachehits
+unbound.total.num.cachemiss,count,,,,summed over threads,1,unbound,total.num.cachemiss
+unbound.total.num.dnscrypt.crypted,count,,,,summed over threads,1,unbound,total.num.dnscrypt.crypted
+unbound.total.num.dnscrypt.cert,count,,,,summed over threads,1,unbound,total.num.dnscrypt.cert
+unbound.total.num.dnscrypt.cleartext,count,,,,summed over threads,1,unbound,total.num.dnscrypt.cleartext
+unbound.total.num.dnscrypt.malformed,count,,,,summed over threads,1,unbound,total.num.dnscrypt.malformed
+unbound.total.num.prefetch,count,,,,summed over threads,1,unbound,total.num.prefetch
+unbound.total.num.zero_ttl,count,,,,summed over threads,1,unbound,total.num.zero_ttl
+unbound.total.num.recursivereplies,count,,,,summed over threads,1,unbound,total.num.recursivereplies
+unbound.total.requestlist.avg,gauge,,,,averaged over threads,0,unbound,total.requestlist.avg
+unbound.total.requestlist.max,gauge,,,,the maximum of the thread requestlist.max values,1,unbound,total.requestlist.max
+unbound.total.requestlist.overwritten,gauge,,,,summed over threads,1,unbound,total.requestlist.overwritten
+unbound.total.requestlist.exceeded,gauge,,,,summed over threads,1,unbound,total.requestlist.exceeded
+unbound.total.requestlist.current.all,gauge,,,,summed over threads,1,unbound,total.requestlist.current.all
+unbound.total.recursion.time.avg,gauge,,,,averaged over threads,0,unbound,total.recursion.time.avg
+unbound.total.recursion.time.median,gauge,,,,averaged over threads,0,unbound,total.recursion.time.median
```

Comment:
Can you provide a better description for the 3 lines above?

Reply:
I'd love to find a good place to put this in the code or docs (or ideally right in metadata.csv, but I don't think that works), but all these descriptions (except a few from older versions that have since disappeared) came from https://nlnetlabs.nl/documentation/unbound/unbound-control/.

The full description for the per-thread recursion.time.median is:
```
              The  median  of  the  time it took to answer queries that needed
              recursive processing.  The median means that  50%  of  the  user
              queries  were  answered  in less than this time.  Because of big
              outliers (usually queries to non responsive servers), the  aver-
              age  can be bigger than the median.  This median has been calcu-
              lated by interpolation from a histogram.
```

Here's a crack at improving the descriptions: https://github.com/DataDog/integrations-extras/pull/451/commits/19a210c071ac7ddc3f9f595ddb9b560a61680780